base/services/wuauserv/wuauserv.c

```diff
@@ -0,0 +1,160 @@
+/*
+*  ReactOS Services
+*  Copyright (C) 2018 ReactOS Team
+*
+*  This program is free software; you can redistribute it and/or modify
+*  it under the terms of the GNU General Public License as published by
+*  the Free Software Foundation; either version 2 of the License, or
+*  (at your option) any later version.
+*
+*  This program is distributed in the hope that it will be useful,
+*  but WITHOUT ANY WARRANTY; without even the implied warranty of
+*  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+*  GNU General Public License for more details.
+*
+*  You should have received a copy of the GNU General Public License along
+*  with this program; if not, write to the Free Software Foundation, Inc.,
+*  51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+*/
+/*
+* COPYRIGHT:        See COPYING in the top level directory
+* PROJECT:          ReactOS Services
+* FILE:             base/services/wuauserv/wuauserv.c
+* PURPOSE:          Windows Update service stub.
+*                   Stub is required for some application at the installation phase.
+* PROGRAMMER:       Denis Malikov
+*/
```

Comment:
Please use the new code header format (see https://www.reactos.org/wiki/Coding_Style ), respecting the whitespace indentation.
Note the presence of the license line, which basically replaces the block of text from above.